theblues/jem.py

```diff
@@ -66,5 +66,5 @@ def get_model(self, macaroons, user, name):
         @param name The name of the model.
         @return The json decoded model.
         """
-        return make_request('{}env/{}/{}'.format(self.url, user, name),
+        return make_request('{}model/{}/{}'.format(self.url, user, name),
```

Comment:
This returns a response including controller-path and controller-uuid, and IIRC that's another difference between v1 and v2. I don't think that's used anywhere and here we just propagate the json returned in the response, so I guess it's ok. Please just check that this assumption is true. It would be nice if these calls created a more explicit python data structure so that we establish a contract and don't just implicitly rely on what's sent over the wire, but that's out of scope for this branch.